bridges/modules/relayers/src/stake_adapter.rs

```diff
@@ -36,7 +36,7 @@ impl<AccountId, BlockNumber, Currency, ReserveId, Stake, Lease>
 	StakeAndSlash<AccountId, BlockNumber, Currency::Balance>
 	for StakeAndSlashNamed<AccountId, BlockNumber, Currency, ReserveId, Stake, Lease>
 where
-	AccountId: Codec + Debug,
+	AccountId: Codec + Debug + PartialEq,
```

Comment:
Why PartialEq instead of Eq here? Isn't reflexive-ness exactly what the intention is here for self comparison?

Reply:
Yea they were not needed. Thanks for the comments!